README.md

```diff
@@ -84,6 +84,22 @@ From the root dir run:
 sudo env PYTHONPATH=. python <file>
 ```
 
+### Docker installation and usage
+
+#### Pre-requisites
+- [Docker](https://docs.docker.com/engine/install/ubuntu/)
+
```

Comment:
Please skip a line between a tag and the text... also, not everyone uses Ubuntu :stuck_out_tongue: 

```suggestion
#### Pre-requisites

- [Docker](https://docs.docker.com/get-docker/)
```